packages/devtools_app/lib/src/service/editor/api_classes.dart

```diff
@@ -21,6 +21,7 @@ enum EditorMethod {
   openDevToolsPage,
 }
 
+/// Method names of LSP requests registered on the Analysis Server.
```

Comment:
nit: any chance we can add a link to where these are defined or to some spec?

Reply:
Yes, done!